src/callbacks/onLayout/initStudyDayControl/startTransition.js

```diff
@@ -0,0 +1,73 @@
+export default function startTransition() {
+    var chart = this;
+    var config = this.config;
+    console.log('Transition Starting');
+
+    function reposition(point) {
+        point
```

Comment:
I think it'd be nice to transition the points themselves to give a sense of the direction.

Reply:
@samussiah Is that not what's happening now? You mean transition to `g` element instead of the `circle` element?